src/test/java/uk/gov/hmcts/ccd/endpoint/userprofile/UserProfileEndpointIT.java

```diff
@@ -452,7 +456,115 @@ public void getAllUserProfiles() throws Exception {
+    public void updateUserProfileForExistingUserAndJurisdiction() throws Exception {
+        // Given the Jurisdiction "TEST3" exists
+        // When attempting to create a User Profile, for an existing user, for the above Jurisdiction
+        UserProfile userProfile = createUserProfile("user4", null, "TEST3", null);
+        Jurisdiction jurisdiction = new Jurisdiction();
+        jurisdiction.setId("TEST3");
+        userProfile.addJurisdiction(jurisdiction);
+        mockMvc.perform(
+            put(CREATE_OR_UPDATE_USER_PROFILE)
+                .contentType(contentType)
+                .content(mapper.writeValueAsBytes(userProfile)))
+            .andExpect(status().is(200));
+
+        // Then no new Jurisdiction is created (the total number remains the same)
+        assertEquals(3,
+            template.queryForObject(COUNT_ALL_JURISDICTIONS_QUERY, Integer.class).intValue());
+
+        // And the User Profile is updated; the user should now belong to two Jurisdictions instead of one
+        final UserProfileEntity userProfileEntity =
+            template.queryForObject(GET_USER_PROFILE_QUERY, this::mapUserProfileData, "user4");
+        assertEquals("user4", userProfileEntity.getId());
+        assertEquals("TEST3", userProfileEntity.getWorkBasketDefaultJurisdiction());
+        assertEquals(2,
+            template.queryForObject(COUNT_USER_PROFILE_ALL_JURISDICTIONS_QUERY, Integer.class, "user4")
+                .intValue());
+    }
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, // checkstyle line break
+        scripts = { "classpath:sql/init_db.sql", "classpath:sql/create_user_profile.sql" })
+    public void updateUserProfileWithSameJurisdiction() throws Exception {
+        // Given the Jurisdiction "TEST2" exists, and the user "user1" already belongs to that Jurisdiction
+        // When attempting to update the User Profile for that user, with Jurisdiction "TEST2"
+        UserProfile userProfile = createUserProfile("user1", null, "TEST2", null);
+        Jurisdiction jurisdiction = new Jurisdiction();
+        jurisdiction.setId("TEST2");
+        userProfile.addJurisdiction(jurisdiction);
+        final MvcResult mvcResult = mockMvc.perform(
+            put(CREATE_OR_UPDATE_USER_PROFILE)
+                .contentType(contentType)
+                .content(mapper.writeValueAsBytes(userProfile)))
+            .andReturn();
+
+        // Then an HTTP 400 (Bad Request) status should be returned
+        assertEquals("Unexpected response status", 400, mvcResult.getResponse().getStatus());
+        assertEquals("Unexpected response message", "User with ID user1 is already a member of the "
+            + "TEST2 jurisdiction", mvcResult.getResponse().getContentAsString());
+    }
+
+    private static UserProfile createUserProfile(final String id,
                                          final String caseType,
```

Comment:
IDE Line identation

Reply:
@adr1ancho for Lines... 564? 568?

Reply:
L568..L579

Reply:
```final String caseType,
final String jurisdiction,
final String state) {
```

Reply:
No big deal if left as it is

Reply:
Ah yes, of course - it _is_ a little untidy... I'll fix :)